CONTRIBUTING.md

```diff
@@ -1,15 +1,16 @@
 # Contributions are welcome!
 
-## Quick guide:
+## Quick guide
 
  * Fork the repo.
+ * Checkout the branch you want to make changes on.
  * Install dependencies: `composer install`.
- * Create branch, e.g. feature-foo or bugfix-bar.
+ * Create branch, e.g. `feature-foo` or `bugfix-bar`.
  * Make changes.
  * If you are adding functionality or fixing a bug - add a test!
  * Fix project itself: `php php-cs-fixer fix`.
- * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify README.rst manually!
- * Check if tests pass: `phpunit`.
+ * Regenerate readme: `php php-cs-fixer readme > README.rst`. Do not modify `README.rst` manually!
+ * Check if tests pass: [`PHPUnit`](https://phpunit.de/manual/current/en/installation.html) (>=4.0)
```

Comment:
before it was a command for CLI, such as `composer install`, `php php-cs-fixer fix` etc. Not it is not.